test/integration/lib/foxboxHelper.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+const fs = require('fs');
+const path = require('path');
+const Config = require('config-js');
+const spawn = require('child_process').spawn;
+
+var config = new Config('./test/integration/lib/config/foxbox.js');
+var FOXBOX_STARTUP_WAIT_TIME_IN_MS = 3000;
+var foxboxInstance;
+
+var helper = (function() {
+
+  function removeUsersDB() {
+    var filePath = path.join(process.env.HOME, 
+      '/.local/share/foxbox/users_db.sqlite'); 
+    fs.unlinkSync(filePath);
```

Comment:
I got this error:
```
  3) Verify validity of pagekite ping endpoint "after all" hook: afterTest:
     Error: ENOENT: no such file or directory, unlink '/home/jlorenzo/.local/share/foxbox/users_db.sqlite'
      at Error (native)
      at Object.fs.unlinkSync (fs.js:932:18)
      at Object.removeUsersDB (test/integration/lib/foxboxHelper.js:16:8)
      at Context.afterTest (test/integration/lib/testPrepper.js:41:19)
```
As `after all` should be responsible for the end state, I don't think we should return an error if no file was initially present. 

I'd recommend to mute the exception in `after()`